next.config.js

```diff
@@ -7,6 +11,84 @@ const nextConfig = {
   }
 }
 
+// API endpoints that should forwarded to the legacy site
+const legacyAPIPaths = [
+  '/total-ogn',
+  '/total-ogv',
+  '/total-ousd',
+  '/circulating-ogn',
+  '/circulating-ogv',
+  '/social-stats',
+  '/mailing-list/:action*',
+]
+
+// Other pages that should be forwarded to the legacy site
+const legacyPagePaths = [
+  // TODO: Do we still use them?
+  '/apple-app-site-association',
+  '/mobile/apk',
+  '/whitepaper.pdf',
+  '/ios',
+  '/android',
+  '/lupefiasco',
+  '/dapp',
+  '/rewards',
+]
+
+// Map legacy routes to new ones here
+const legacyPageMappings = [
+  // [source, destination]
+  ['/dashboard', '/'],
+  ['/presale', '/'],
+  ['/tokens', '/'],
+  ['/investors', '/'],
+  
+  ['/product', '/'],
+  ['/ogn-token', '/'],
+  
+  ['/team', '/community'],
+  ['/about', '/community'],
+
+  ['/discord', `${NEXT_PUBLIC_DISCORD_URL}`],
+  ['/telegram', `${NEXT_PUBLIC_TELEGRAM_URL}`],
+
+  // TODO: Do we still use them?
+  ['/admin', 'https://admin.staging.originprotocol.com'],
+]
+
+// TODO: Check these missing endpoints from legacy site with someone
+// - /whitepaper
```

Comment:
All this content is outdated. 
I'd suggest we serve the new homepage for all those
CC @micahalcorn